### PR TITLE
Taught import-task to extract into current-working-dir.

### PR DESCRIPTION
**Pull request by ggainey**
fixes #8610
[nocoverage]

### CODE REVIEW

**Review by ipanova — APPROVED** [review]

File: CHANGES/8610.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+PulpImporter now unpacks into the task-worker's working directory rather than /tmp.
```
Comment (mdellweg) [inline CHANGES/8610.bugfix:1]: Optional: Briefly describe the kind of trouble this caused that admins can recognize.

**Review by mdellweg — APPROVED** [review]

File: CHANGES/8610.bugfix (hunk 1 above)
Comment (ggainey) [reply CHANGES/8610.bugfix]: :+1:

**Review by bmbouter — APPROVED** [review]
Nice

**Comment by ggainey** [comment]
> ### Backport to 3.17:  backport PR created
> white_check_mark Backport PR branch: `patchback/backports/3.17/2f565c936d03f26c61c4b992c5e98a2b835d7cc7/pr-1764`
> 
> Backported as #2249
> 
> ##
> robot @patchback I'm built with [octomachinery](https://octomachinery.dev) and my source is open — https://github.com/sanitizers/patchback-github-app.

3.17 label added in error - the original PR was included in the 3.17 release.

**Pull request merged**